### PR TITLE
Will terminate job if illegal fil-server is used.

### PR DESCRIPTION
**Pull request by joakim-hove**
**Task**
This is very Statoil specific. If a forward model simulation is found to be started from a "illegal" file server the whole job can be terminated with a message to the user.

**Approach**
Send an e-mail to the user and `sys.exit(1)`

**Pre un-WIP checklist**
- [ ] Statoil tests pass locally
- [x] Have completed graphical integration test steps

### CODE REVIEW

File: bin/job_dispatch.py [hunk 1]
```diff
@@ -17,8 +17,11 @@
 import subprocess
 import shutil
 import datetime
+import smtplib
 from email.mime.text import MIMEText
 
+block_illegal_fileserver = False
```
Comment (markusdregi) [inline bin/job_dispatch.py]: Please move this assignment below the import "section"

Comment (joakim-hove) [reply bin/job_dispatch.py]: OK - done

File: bin/job_dispatch.py [hunk 2]
```diff
@@ -17,19 +17,37 @@
...
 ERROR_URL     = "http://st-vlinbuild01.st.statoil.no:8000/api/error/"
 
 
+def illegal_fileserver_exit(msg_txt , user):
+    from_ = "no-reply@statoil.com"
+    to = "%s@statoil.com" % user
+    msg = MIMEText( msg_txt )
+    msg['to'] = to
+    msg['from'] = from_
+    msg['subject'] = "Illegal fil-server"
```
Comment (markusdregi) [inline bin/job_dispatch.py:42]: `fil-server` -> `file server`

**Review by markusdregi — APPROVED** [review]
When the minor typo in the message subject is fixed, this one is good to go.

I would like us to in the future restructure this so that the Statoil specifics are removed from github.

**Pull request merged**